web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -93,7 +93,7 @@ context('Create Route with Upstream', () => {
     cy.contains(data.routeName).siblings().contains('Edit').click();
 
     // fetching Resource
-    cy.wait(500);
+    cy.get(domSelector.name).click();
```

Comment:
click what?

Reply:
Why click a name?

Reply:
During the test, I found that the new url can be resolve directly after clicking

Reply:
We need to be very clear about the reason for each line of code.